data/helpers.d/mysql

```diff
@@ -90,3 +90,38 @@ ynh_mysql_create_user() {
 ynh_mysql_drop_user() {
     ynh_mysql_execute_as_root "DROP USER '${1}'@'localhost';"
 }
+
+# Create a database, an user and its password. Then store the password in the app's config
+#
+# User of database will be store in db_user's variable.
+# Name of database will be store in db_name's variable.
+# And password in db_pwd's variable.
+#
+# usage: ynh_mysql_generate_db user name
+# | arg: user - Proprietary of the database
+# | arg: name - Name of the database
+ynh_mysql_generate_db () {
+	db_user=${1//[-.]/_}	# Mariadb doesn't support - and . in the name of databases. It will be replace by _
+	db_name=${2//[-.]/_}
```

Comment:
It would be better to just fail (because it's a programming error). With this implementation, the caller does not know how the provided user/name is modified, so if he uses it afterwards (in a conf file), this will fail